cli/cli_account.go

```diff
@@ -68,10 +68,19 @@ func (c *cmd) Account(ctx *cli.Context) error {
 
 	if err := displayServiceStatus("Dedicated IP",
 		payload.DedicatedIpStatus,
-		payload.DedicatedIpExpiresAt); err != nil {
+		payload.LastDedicatedIpExpiresAt); err != nil {
 		return err
 	}
 
+	mfa := "enabled"
```

Comment:
Would be good to have constants defined and used on both ends

Reply:
If you are referring on passing the string `"enabled"` in the gRPC response, then I think it is better to pass the enum instead of strings, as it is now. 
Because the string is only useful if it is displayed as it is, otherwise will be string comparison vs int comparison. In that case I think int comparison is nicer :smiley: .